component/src/main/java/io/siddhi/extension/io/http/sink/SSEServerSink.java

```diff
@@ -70,36 +70,14 @@
 /**
  * Http sink to publish sse events.
  */
-@Extension(name = "sse", namespace = "sink",
+@Extension(name = "sse-server", namespace = "sink",
```

Comment:
@ramindu90 we have to change async api generation name right if this changed ?

Reply:
@dnwick yes